mangaki/mangaki/wrappers/anilist.py

```diff
@@ -97,6 +239,24 @@ def _request(self,
         r.raise_for_status()
         return r.json()
 
+    def list_seasonal_animes(self,
```

Comment:
It's indeed way better to read when calling the method 👍 ! Didn't know we could enforce keyword arguments